tests/tree_algorithm.py

```diff
@@ -0,0 +1,180 @@
+from input_output.file_output import load_graph_list, create_csv_file, write_csv_line
+from input_output.sys_output import fail, passed
+from algorithms.tree_algorithm import trees_count_isomorphisms
+from algorithms.preprocessing import fix_degrees
+from time import time
+
+
+"""
+To test if the tree_algorithm works finding and counting isomorphisms of trees.
+"""
+
+
+def unit_test(write_csv_any=False, write_stdout_passed=True, write_stdout_fail=True):
+    test_name = 'tree_algorithm'
+    if write_csv_any:
+        csv_filepath = create_csv_file(test_name)
+        print('<' + test_name + '> ' + 'Appending to CSV: ' + "file:///" + csv_filepath.replace('\\', '/') + '\nStart...')
+
+        write_csv_line(csv_filepath, ['file', 'i', 'j', 'Pass?', 'Are isomorph Time (s)', 'Amount isomorph Time (s)'])
+    else:
+        print('<' + test_name + '> ')
+
+    # TEST FILE
```

Comment:
Wat mij betreft kan deze lijst wel worden aangevuld met de testen uit tests/branching.py
[De algoritmes in die test zijn toch niet snel genoeg]

```
    # TEST FILE
    files = ["bigtrees1", "bigtrees3", "trees90", "trees36"]
    solution = [
        {(0, 2): 442368, (1, 3): 5308416},
        {(0, 2): 2772351862699137701073289910157312, (1, 3): 462058643783189616845548318359552},
        {(0, 3): 6912, (1, 2): 20736},
        {(0, 7): 2, (1, 4): 6, (2, 6): 2, (3, 5): 6}
    ]
```

We kunnen overwegen of we de tree test graphs kunnen schrappen uit tests/branching.py